addOns/oast/src/main/java/org/zaproxy/addon/oast/services/interactsh/InteractshService.java

```diff
@@ -156,8 +156,7 @@ public void optionsChanged(OptionsParam optionsParam) {
                         "Error during applying Interactsh config changes - server URL: {} and polling frequency: {} seconds. {}",
                         param.getServerUrl(),
                         param.getPollingFrequency(),
-                        e.getMessage(),
-                        e);
+                        e.getMessage());
```

Comment:
I guess the stack trace could be logged at debug.

Reply:
IMO if it's a rare / unexpected error I think the stack trace should still be logged. If a user runs into this and shares details with us, a stack trace is essential to debug the issue.